.travis.yml

```diff
@@ -0,0 +1,30 @@
+language: python
+python:
+  - 3.4
+  - 3.5
+  - 3.6
+
+cache:
+  pip: true
+
+before_install:
+  # Use pip 9, so that setuptools can be downgraded.
+  - pip install pip==9 setuptools==17
```

Comment:
Tried twice after removing this line, failing on Python 3.4 https://travis-ci.org/satwikkansal/coala-quickstart/jobs/242928742